### DISCUSSION TITLE
`ctrl+m` sends `09;5u` instead of executing the command

### DISCUSSION THREAD

**Discussion by aslushnikov** [ghostty-org/ghostty#3586] [Feature Requests, Ideas] [answered]
Hi! 

In all terminals I've used, `ctrl+m` is by default mapped to a newline. However, in Ghostty it results in `09;5u` code being inserted.
This shortcut works in all terminals I work with and in VIM; i'd love to have the same in Ghostty. I tried setting up a custom key binding to achieve this behavior, but failed to do so.

1. Is it possible to have `ctrl+m` as a newline by default?
2. If not, is there a way to achieve this?

Thanks for your time!
- Andrey

**Comment by JakeWharton** [comment, marked as answer]
Check https://github.com/ghostty-org/ghostty/discussions/3151#discussioncomment-11678064

This explains the current behavior and how to fix it. You will use your key combo as well as `\n` for the text instead.
